en/sim_gazebo_gz/tools_avl_automation.md

```diff
@@ -0,0 +1,162 @@
+# Advanced Lift Drag (AVL) Automation Tool
+
+The Gazebo [Advanced Plane](../sim_gazebo_gz/vehicles.md#advanced-plane) vehicle model uses the _Advanced Lift Drag_ plugin to model vehicle lift and drag behaviour.
+This tool allows you to calculate the parameters required to create a _Advanced Lift Drag_ plugin for your own particular vehicle.
+
+You only have to provide a few parameters for each wing foil and the tool will use this information to call the Athena Lattice Vortex (AVL) to make the necessary calculations.
+The results will then automatically be written into a provided plugin template that can then be copy-pasted into a model or world sdf file.
+
+## Install
+
+To setup the tool:
+
+1. Download AVL 3.36 from <https://web.mit.edu/drela/Public/web/avl/>.
+   The file for AVL version 3.36 can be found about halfway down the page.
+1. After downloading, extract AVL and move it to the home directory using:
+
+   ```sh
+   sudo tar -xf avl3.36.tgz
+   mv ./Avl /home/
+   ```
+
+1. Follow the **README.md** found in `./Avl` to finish the setup process for AVL (this requires that you set up `plotlib` and `eispack` libraries).
+   We recommend using the `gfortran` compile option, which might further require that you to install `gfortran`.
+   On Ubuntu can be done by running:
+
+   ```sh
+   sudo apt update
+   sudo apt install gfortran
+   ```
+
+   When running the Makefile for AVL, you might encounter an `Error 1` message stating that there is a directory missing.
+   This does not prevent AVL from working for our purposes.
+
+Once the process described in the AVL README is completed, AVL is ready to be used.
+No further set up is required on the side of the AVL or the tool.
+
+If you want to move the location of the AVL directory, this can simply be done by passing the `--avl_path` flag to the `input_avl.py` file, using the desired directory location for the flag (don't forget to place a "/" behind the last part of the path).
+Running this will automatically also adjust the paths where necessary.
+
+## Run AVL
+
+An example template has been provided in the form of the `input.yml` that implements a standard plane with two ailerons, an elevator and a rudder.
+This example template can be run using: `python input_avl.py --yaml_file input.yml`.
+
+To run the tool for your plane:
+
+1. Copy the example `input.yml` to `<your_custom_yaml_file>.yml` and modify it to match your desired plane
+1. Run the tool on your yml file:
+
+   ```sh
+   python input_avl.py <your_custom_yaml_file>.yml
+   ```
+
+   Note that the `yaml` and `argparse` packages must be present in your Python environment.
+
+1. The tool prompts for a range of vehicle specific parameters that are needed in order to specify the geometry and physical properties of the plane.
```

Comment:
I pulled this bit out of "Functionality" as it is part of "what you do as a user". The functionality bit is now purely "what the tool does". It is good to separate like this.